solutions/dashboard/deployment/README.md

```diff
@@ -42,6 +42,17 @@ Username: admin
 Password:
 'admin:login' logged in successfully
 ```
+- Create ElasticSearch Application using CLI
+```
+argocd app create elasticsearch \
+--repo https://github.com/shailesh-vaidya/cortx-re \
```

Comment:
is it going to be your fork?

Reply:
Yes. Till we fully migrate to Argo CD for deployment